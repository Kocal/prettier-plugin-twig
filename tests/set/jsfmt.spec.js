run_spec(__dirname, ["twig"]);
run_spec(__dirname, ["twig"], { singleQuote: true });
