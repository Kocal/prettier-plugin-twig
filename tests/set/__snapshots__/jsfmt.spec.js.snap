// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`set.twig 1`] = `
====================================options=====================================
parsers: ["twig"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% set foo = 'bar' %}
{%  set  foo  =     'bar'   %}
{%- set foo = 'bar' -%}
{%-    set     foo = 'bar'-%}
{%  set foo, bar= 'bar',  'foo'  %}
{% set foo = 'with a "' %}
{% set foo = "with a '" %}
{% set foo = [  5,   2 , 3    ]  %}

=====================================output=====================================
{% set foo = "bar" %}
{% set foo = "bar" %}
{% set foo = "bar" %}
{% set foo = "bar" %}
{% set foo, bar = "bar", "foo" %}
{% set foo = 'with a "' %}
{% set foo = "with a '" %}
{% set foo = [5, 2, 3] %}

================================================================================
`;

exports[`set.twig 2`] = `
====================================options=====================================
parsers: ["twig"]
printWidth: 80
singleQuote: true
                                                                                | printWidth
=====================================input======================================
{% set foo = 'bar' %}
{%  set  foo  =     'bar'   %}
{%- set foo = 'bar' -%}
{%-    set     foo = 'bar'-%}
{%  set foo, bar= 'bar',  'foo'  %}
{% set foo = 'with a "' %}
{% set foo = "with a '" %}
{% set foo = [  5,   2 , 3    ]  %}

=====================================output=====================================
{% set foo = 'bar' %}
{% set foo = 'bar' %}
{% set foo = 'bar' %}
{% set foo = 'bar' %}
{% set foo, bar = 'bar', 'foo' %}
{% set foo = 'with a "' %}
{% set foo = "with a '" %}
{% set foo = [5, 2, 3] %}

================================================================================
`;
